moped-editor/src/views/projects/projectView/ProjectView.js

```diff
@@ -204,7 +208,6 @@ const ProjectView = () => {
    */
   const { loading, error, data, refetch } = useQuery(SUMMARY_QUERY, {
     variables: { projectId, userId },
-    fetchPolicy: "no-cache",
   });
```

Comment:
even though there are parts of this query that can be cached (the lookup values for project_phases, etc) — that cache is not going to stop us from re-querying the DB, because they're bundled together with the project data that we always want to be current.

so we have a backlog task of breaking apart some of our queries so that we can fetch the lookup tables once on app load. we should never need to fetch them more than once per user session. i will open an issue for that 😅

Reply:
Good to finally see a path to fixing the repeated queries for those lookups! That will be huge. 🙌